src/kafkaaggregator/config.py

```diff
@@ -71,27 +73,60 @@ class Configuration:
     workload of the application.
     """
 
-    src_topic: str = os.getenv("SOURCE_TOPIC", "kafkaaggregator-src-topic")
+    source_topic_name: str = os.getenv(
+        "SOURCE_TOPIC", "kafkaaggregator-example"
+    )
     """Name of the source topic used in the kafkaaggregator example."""
 
-    agg_topic: str = os.getenv(
-        "AGGREGATION_TOPIC", "kafkaaggregator-agg-topic"
+    topic_regex: str = os.getenv("TOPIC_REGEX", "^kafkaaggregator-example$")
+    """Regex to select topics to aggregate."""
+
+    topic_blacklist: List[str] = field(default_factory=list)
+    """Any topic names added to the topic_blacklist will not be aggregated."""
+
+    topic_rename_format: str = os.getenv(
+        "TOPIC_RENAME_FORMAT", "{source_topic_name}-aggregated"
     )
-    """Name of the aggregation topic used in the kafkaaggregator example."""
+    """A format string for the aggregation topic name, which must contain
+    ``{source_topic_name}`` as a placeholder for the source topic name."""
```

Comment:
Normally in multiline docstrings we put the last `"""` on a newline.

Reply:
It passed `flake8-docstrings` though.

Reply:
Thanks for pointing that I've just read [PEP 257](https://www.python.org/dev/peps/pep-0257/) and for a multi-line docstring I should really have a one-line summary line, followed by a blank line and a more elaborate description. I have seen `flake8-docstring` identifying those problems and now I'm curious why it failed so bad here.

Reply:
I've tested that in a function and in a data class method and `fake8-docstrings` identifies those problems correctly. However, it seems that it does not check docstrings in data class fields. That seems to be the problem.

```
flake8...................................................................Failed
- hook id: flake8
- exit code: 1

src/kafkaaggregator/config.py:106:1: D205 1 blank line required between summary line and description
src/kafkaaggregator/config.py:106:1: D209 Multi-line docstring closing quotes should be on a separate line
src/kafkaaggregator/config.py:106:1: D400 First line should end with a period
```

Reply:
OK, looks like PEP-258 suggested adding [attribute docstrings](https://www.python.org/dev/peps/pep-0258/#attribute-docstrings), but it was rejected. It seems that there's no standard for data class field docstrings and that's why `flake8-docstrings` ignores those.